interview-questions.md

```diff
@@ -8,48 +8,54 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
-
-Researched answer:
+Your answer: Javascript Syntax eXchange? I forget what the acronym means but it's what allows javascript to play with HTML's toys.
 
+Researched answer: Dang, I was close. The acronym is Javascript Syntax eXtension, or just Javascript XML. An interesting Syntactical difference would be the need to dance around Javascript's reserved keywords, like how "for" in HTMLspeak would be "HTMLfor".
+***something about the phrase "syntactic sugar" really irks me...***
 1. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: A package manager, similar to NPM, that we use to install/run React and Jest.
+
+***yarn is a long continuous length of interlocked fibres, used in sewing, crocheting, knitting, weaving, embroidery, ropemaking, and the production of textiles. Yarn is the package manager. gotta watch that Capitalization!***
 
-Researched answer:
+Researched answer: running [yarn] in my terminal gives me the yarn.lock file and a node_modules folder.
```

Comment:
I feel like this is a bit to the point and would lead an interviewer down the rabbit hole of questions. How could this be a little more elaborated on?